centos2almaconverter/actions/postgres.py

```diff
@@ -18,6 +19,42 @@ def _do_check(self):
         return not postgres.is_postgres_installed() or not postgres.is_database_initialized() or not postgres.is_database_major_version_lower(_ALMA8_POSTGRES_VERSION)
 
 
+class AssertPostgresLocaleMatchesSystemOne(action.CheckAction):
+    def __init__(self):
+        self.name = "checking if system locale is safe for Postgres databases upgrade"
+        self.description = """Postgres database upgrade expects system locale to match the one databases were created with.
+\tYou may need to change system locale (see /etc/locale.conf and the locale command) or
```

Comment:
NIT: It could be helpful to display system and Postgres locales in the message

Reply:
These can be seen in the log. I think that should suffice.